.gitignore

```diff
@@ -0,0 +1,2 @@
+__pycache__
+JIRAAnnouncer.egg-info
```

Comment:
If adding .gitignore, should use Pyramid standard ignore:
*.egg
*.egg-info
*.pyc
*$py.class
*~
.coverage
coverage.xml
build/
dist/
.tox/
nosetests.xml
env*/
tmp/
Data.fs*
*.sublime-project
*.sublime-workspace
.*.sw?
.sw?
.DS_Store
coverage
test
venv/

Reply:
Updated